### PR TITLE
Formatted Changes for Github Actions

### PR DESCRIPTION
**Pull request by Mustaballer**
## Overview

- Formatted files from running `python black .` and for docstrings in Google format
- Reference Github Actions PR: https://github.com/MLDSAI/OpenAdapt/pull/215

## Unit Tests Created

- n/a

## Steps to QA

- See Github Actions run history on my repo: https://github.com/Mustaballer/puterbot/actions

### CODE REVIEW

File: alembic/versions/104d4a614d95_add_performancestat.py [hunk 1]
```diff
@@ -10,27 +10,30 @@
 
 
 # revision identifiers, used by Alembic.
-revision = '104d4a614d95'
-down_revision = 'b2dc41850120'
+revision = "104d4a614d95"
```
Comment (abrichr) [inline alembic/versions/104d4a614d95_add_performancestat.py:13]: @Mustaballer can you please modify the configuration such that files in this directory are ignored? They are auto-generated and can stay as-is.

Comment (Mustaballer) [reply alembic/versions/104d4a614d95_add_performancestat.py:13]: Yeah sure will do :)

**Comment by abrichr** [comment]
@Mustaballer can you please fix merge conflicts? 🙏

**Comment by abrichr** [comment]
@Mustaballer can you please fix merge conflicts? 🙏

**Comment by Mustaballer** [comment]
Done :)

**Comment by abrichr** [comment]
@Mustaballer thank you! I see this is marked as a draft, what else needs to be done before we can merge it?

File: openadapt/cache.py [hunk 2]
```diff
@@ -9,18 +9,55 @@
...
         def wrapper(*args, **kwargs):
+            """
+
+            Args:
+              *args: 
+              **kwargs: 
+
+            Returns:
+
+            """
```
Comment (abrichr) [inline openadapt/cache.py:60]: Was this generated automatically? If so, can you please clarify what generated it? I'm not sure it's helpful.

Comment (Mustaballer) [reply openadapt/cache.py:60]: I used `pyment -w -o google .` which automatically generates doctstrings in google python style.

**Comment by Mustaballer** [comment]
I will close this PR and address formatting changes in a new one.

**Pull request closed without merging**